stubs/tests/Feature/AuthenticationTest.php

```diff
@@ -1,63 +1,52 @@
 <?php
 
-namespace Tests\Feature;
-
 use App\Models\User;
 use Illuminate\Foundation\Testing\RefreshDatabase;
-use Tests\TestCase;
-
-class AuthenticationTest extends TestCase
-{
-    use RefreshDatabase;
-
-    public function test_login_screen_can_be_rendered()
-    {
-        $response = $this->get(localized_route('login'));
-
-        $response->assertOk();
-    }
-
-    public function test_users_can_authenticate_using_the_login_screen()
-    {
-        $user = User::factory()->create();
-
-        $response = $this->post(localized_route('login'), [
-            'email' => $user->email,
-            'password' => 'password',
-        ]);
-
-        $this->assertAuthenticated();
-        $response->assertRedirect(localized_route('dashboard'));
-    }
-
-    public function test_users_can_not_authenticate_with_invalid_password()
-    {
-        $user = User::factory()->create();
-
-        $this->post(localized_route('login'), [
-            'email' => $user->email,
-            'password' => 'wrong-password',
-        ]);
-
-        $this->assertGuest();
-    }
-
-    public function test_users_can_edit_own_profiles()
-    {
-        $user = User::factory()->create();
-
-        $response = $this->post(localized_route('login'), [
-            'email' => $user->email,
-            'password' => 'password',
-        ]);
-
-        $response = $this->get(localized_route('users.edit'));
-        $response->assertOk();
-    }
-
-    public function test_guests_can_not_edit_profiles()
-    {
-        $response = $this->get(localized_route('users.edit'));
-        $response->assertRedirect(localized_route('login'));
-    }
-}
+
+uses(RefreshDatabase::class);
+
+test('login screen can be rendered', function () {
+    $response = $this->get(localized_route('login'));
+
+    $response->assertOk();
+});
+
+test('users can authenticate using the login screen', function () {
+    $user = User::factory()->create();
+
+    $response = $this->post(localized_route('login'), [
+        'email' => $user->email,
+        'password' => 'password',
+    ]);
+
+    $this->assertAuthenticated();
```

Comment:
Are you going to update these to use the new Pest syntax like `assertAuthenticated();`?

Reply:
No, I don't think so. I ran `vendor/bin/pest --drift` but I don't think it's worth spending time on that.